cpp/include/functions.h

```diff
@@ -182,22 +182,6 @@ void delete_edge_list(Graph* graph);
 /* ----------------------------------------------------------------------------*/
 void delete_transposed_adj_list(Graph* graph);
 
-/**
- * @Synopsis   Find pairs of vertices in the input graph such that each pair is connected by
- *             a path that is two hops in length.
- *
- * @param[in]* graph                 in  : graph descriptor with graph->adjList pointing to a gdf_adj_list structure
- *
- * @param[out] first                 out : An uninitialized gdf_column which will be initialized to contain the
- *                                         first entry of each result pair.
- * @param[out] second                out : An uninitialized gdf_column which will be initialized to contain the
- *                                         second entry of each result pair.
- *
- * @throws     cugraph::logic_error when an error occurs.
- */
-/* ----------------------------------------------------------------------------*/
-void get_two_hop_neighbors(Graph* graph, gdf_column* first, gdf_column* second);
```

Comment:
NOTE:  I made get_two_hop_neighbors an algorithm.  Seemed more like an algorithm than a utility.

Reply:
it does need to be redone for better performance (https://github.com/rapidsai/cugraph/issues/245)